dev-games/ogre/metadata.xml

```diff
@@ -91,8 +91,9 @@ Exporters
         not be enabled unless an application really needs it.
     </flag>
     <flag name="freeimage">Support images via media-libs/freeimage</flag>
-    <flag name="gl3plus">Build OpenGL 3+ RenderSystem (EXPERIMENTAL)</flag>
+    <flag name="gl3plus" restrict="~dev-games/ogre-1.9.0">Build OpenGL 3+ RenderSystem (EXPERIMENTAL)</flag>
     <flag name="gles2" restrict="~dev-games/ogre-1.9.0">Build OpenGL ES 2.x RenderSystem</flag>
+    <flag name="gles2" restrict="~dev-games/ogre-1.12.9">Build OpenGL ES2/3 RenderSystem</flag>
```

Comment:
Can't we just write 'restrict="&lt;dev-games/ogre-2.0.0' instead of duplicating flag entries?

Reply:
Actually it seems ogre-2.x also support gles2 rendering system, probably you should also add gles2 support to ogre-2.x ebuild then these restricts are not needed at all.

Reply:
Yes, yes... of course, you are right.

Reply:
I guess I got caught up in the "ES 2.x" versus "ES2/3" difference. Maybe just write "OpenGL ES 2+" instead?

Reply:
I noticed there was a 'gles3' flag in older ogre version. Do you think we should follow the convention and make a separate gles3 flag?

Reply:
Actually I would like to see ogre-1.9.0 get dumped. But yes, that is what came to my mind, too.
GLES2/3 are separated in Ogre-1.9, while it is unified since ogre-1.10.
Well, the gles2 USE flag is disabled in Ogre-2.1, because that one isn't ported, yet.
So I came to the conclusion, as those USE flag descriptions should be left in place as they are. At least for now.

I also doubt that it ever will, because main development is with Ogre-2.2
v2.1 came out in april, while v2.2 had five release since then.

Once we have sorted out Ogre-1.12.9, I'd like to have another look at v2.2. Maybe we should ditch v2.1 altogether, like we did with v2.0.

Clarification: The v2.0 branch was a hybrid, used to push forward the move from CPU centric (v1.x) to GPU powered (v2.x) rendering. The latter is clearly the thing, but right now it looks to me like v2.1 is a transition stage like v2.0 was.